src/tooltip/tooltip.js

```diff
@@ -339,6 +347,72 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
   }];
 })
 
+// This is mostly ngInclude code but with a custom scope
+.directive( 'tooltipTemplateTransclude', [
```

Comment:
It's a little hard to generalize though, because I don't want to create another scope here (notice the directive only has a link function), and it specifically looks for `origScope` and `content` to be present on the scope.